contracts/zero-ex/tests/utils/DeployZeroEx.sol

```diff
@@ -0,0 +1,304 @@
+        emit log_named_address("UniswapFeature", address(ZERO_EX_DEPLOYED.features.uniswapFeature));
+        emit log_named_address("FundRecoveryFeature", address(ZERO_EX_DEPLOYED.features.fundRecoveryFeature));
+        emit log_named_address("MetaTransactionsFeature", address(ZERO_EX_DEPLOYED.features.metaTransactionsFeature));
+        emit log_named_address("ERC1155OrdersFeature", address(ZERO_EX_DEPLOYED.features.erc1155OrdersFeature));
+        emit log_named_address("ERC721OrdersFeature", address(ZERO_EX_DEPLOYED.features.erc721OrdersFeature));
+        emit log_named_address("TransformERC20Feature", address(ZERO_EX_DEPLOYED.features.transformERC20Feature));
+        emit log_named_address("MultiplexFeature", address(ZERO_EX_DEPLOYED.features.multiplexFeature));
+
+        emit log_string("----- Transformers -----");
+        emit log_named_address("FillQuoteTransformer", address(ZERO_EX_DEPLOYED.transformers.fillQuoteTransformer));
+        emit log_named_address("WethTransformer", address(ZERO_EX_DEPLOYED.transformers.wethTransformer));
+        emit log_named_address(
+            "AffiliateFeeTransformer",
+            address(ZERO_EX_DEPLOYED.transformers.affiliateFeeTransformer)
+        );
+        emit log_named_address("PayTakerTransformer", address(ZERO_EX_DEPLOYED.transformers.payTakerTransformer));
+
+        emit log_string("----- Other -----");
+        emit log_named_address("WETH", address(ZERO_EX_DEPLOYED.weth));
+    }
+
+    function deployZeroEx() public returns (ZeroExDeployed memory) {
+        if (isDeployed) {
+            return ZERO_EX_DEPLOYED;
+        }
+
+        ZERO_EX_DEPLOYED.weth = IEtherTokenV06(address(new WETH9V06()));
+        InitialMigration initialMigration = new InitialMigration(address(this));
+        // Append the required ZeroEx constructor arguments (address bootstrapper)
+        bytes memory zeroExDeploycode = abi.encodePacked(type(ZeroEx).creationCode, abi.encode(initialMigration));
+        // Append the required deployer code constructor arguments (bytes initCode)
+        bytes memory deployerDeploycode = abi.encodePacked(deployerBytecode, abi.encode(zeroExDeploycode));
+        // HERE BE DRAGONS, feel free to ignore this for now
+        // We want to deploy the ZeroEx contract, at 0xDef1C0ded9bec7F1a1670819833240f027b25EfF
+
+        // We use a special mechanism to deploy the ZeroEx contract.
+        // Must occur from this address as the first transaction
+        // The address is technically emitted in an event, but we know we did it correctly
+        //│   │   ├─  emit topic 0: 0xf40fcec21964ffb566044d083b4073f29f7f7929110ea19e1b3ebe375d89055e
+        //│   │   │           data: 0x000000000000000000000000def1c0ded9bec7f1a1670819833240f027b25eff
+        hoax(VANITY_DEPLOYER_ADDRESS);
+        assembly {
+            pop(create(0, add(deployerDeploycode, 0x20), mload(deployerDeploycode)))
```

Comment:
I admit I haven't used inline assembly in solc 0.6, but isn't the `pop` here redundant?